<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionDechet</class>
 <widget class="QDialog" name="GestionDechet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>632</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Gestion Dechets</string>
  </property>
  <widget class="QTabWidget" name="tabWidget2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1261</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_17">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Matricule de camion&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Le nom de cité qu'il va etre nettoyer :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Le temps départ :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter1">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matricule">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>170</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cite">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_quitter">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>260</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matricule_D">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Matricule &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="le_date">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>310</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTimeEdit" name="le_temps">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1221</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
    <zorder>pb_ajouter1</zorder>
    <zorder>le_matricule</zorder>
    <zorder>le_cite</zorder>
    <zorder>pb_quitter</zorder>
    <zorder>le_matricule_D</zorder>
    <zorder>label_18</zorder>
    <zorder>le_date</zorder>
    <zorder>le_temps</zorder>
   </widget>
   <widget class="QWidget" name="tab_18">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tab_dechet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>671</width>
       <height>381</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_88">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>490</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; color:#ffffff;&quot;&gt;Supprimer un groupe (par destination) :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_supp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>231</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supp">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>540</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_recherche">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>301</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>330</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Date :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;L’heure du départ :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_cite_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>290</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>250</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Matricule du camion :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>290</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Le nom de la cité qui va être nettoyée :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matricule_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>250</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_MODIFIER">
     <property name="geometry">
      <rect>
       <x>970</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_recherche">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MATRICULE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MATRICULED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESTINATION</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TEMPS_A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DATE_A</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pb_tri_dest">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>trier </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_matricule_D2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>180</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>180</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Matricule  :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_121">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Rechercher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_41">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>90</y>
       <width>20</width>
       <height>391</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_42">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>470</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_43">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>80</y>
       <width>481</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="ordre_tri_dechet">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ASC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESC</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="colone_tri_dechet">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>MATRICULED</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DESTINATION</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="le_date_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>330</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTimeEdit" name="le_temps_2">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>210</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1221</width>
       <height>611</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>tab_dechet</zorder>
    <zorder>label_88</zorder>
    <zorder>le_supp</zorder>
    <zorder>pb_supp</zorder>
    <zorder>le_recherche</zorder>
    <zorder>label_19</zorder>
    <zorder>label_21</zorder>
    <zorder>le_cite_2</zorder>
    <zorder>label_17</zorder>
    <zorder>label_20</zorder>
    <zorder>le_matricule_2</zorder>
    <zorder>pb_MODIFIER</zorder>
    <zorder>cb_recherche</zorder>
    <zorder>pb_tri_dest</zorder>
    <zorder>le_matricule_D2</zorder>
    <zorder>label_22</zorder>
    <zorder>label_121</zorder>
    <zorder>line_41</zorder>
    <zorder>line_42</zorder>
    <zorder>line_43</zorder>
    <zorder>ordre_tri_dechet</zorder>
    <zorder>colone_tri_dechet</zorder>
    <zorder>le_date_2</zorder>
    <zorder>le_temps_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
